docs/conf.py

```diff
@@ -147,29 +156,38 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, 'tutorials.tex', u'Tutorials Documentation',
-     u"Juliette Zito", 'manual'),
-]
+latex_documents = [(
```

Comment:
Everything below falls under the umbrella of https://github.com/nlesc-nano/Tutorials/pull/3#discussion_r428649879.